src/PPU.cpp

```diff
@@ -273,23 +276,26 @@ void PPU::updateTileMaps() {
 	bgp[2] = (bgpData & 0x30) >> 4;
 	bgp[3] = (bgpData & 0xC0) >> 6;
 
-	// Iterate through 32x32 tiles for the background and window
-	for (int i = 0; i < 0x0400; i++) {
-		uint8_t x = i % 32;
-		uint8_t y = i / 32;
+	for (int i = 0; i < 32; i++) {
+		uint8_t x = i * 8;
```

Comment:
with two loops it seems more readable, and also maybe faster since I don't have to do the * 8 calculation as much. Not sure if its offset by more loop overhead, or if it even matters.

Reply:
Also these aren't being used currently anyway so it's fairly moot.